### PR TITLE
Replace elastic containers with the ones maintained by Bitergia

### PR DESCRIPTION
**Pull request by sanacl**
Replace customized elastic containers with the ones maintained by the Bitergia team. This includes the scripts needed to set the credentials of the Elasticsearch database + Kibana.

### CODE REVIEW

File: docker-compose-build.yml [hunk 1]
```diff
@@ -136,30 +136,37 @@ services:
...
+        image: bitergia/elasticsearch:6.1.0-secured
         command: /elasticsearch/bin/elasticsearch -E network.bind_host=0.0.0.0 -Ehttp.max_content_length=500mb
         networks:
             - default
         expose:
             - 9200
         ports:
           - "9200:9200"
+        volumes:
+          - /elasticsearch-index/:/elasticsearch/data
```
Comment (MarcioMateus) [inline docker-compose-build.yml]:
```suggestion
          - ~/elasticsearch-index/:/elasticsearch/data
```
We should avoid trying to mount volume on filesystem root.

**Comment by sanacl** [comment]
> After `docker-compose up`, when I run `cd elasticsearch-sg && ./add_user.sh kibana thekibanauser thekibanapassword` I got the following error:
> 
> ```
> >>> File generated at sgconfig/sg_internal_users.yml adding the user thekibanauser
> 
> Search Guard Admin v6
> Will connect to localhost:9300
> ERR: Seems there is no Elasticsearch running on localhost:9300 - Will exit
> ```

If you are deploying this in a machine where ES was not running you'll need to execute this `sudo sysctl -w vm.max_map_count=262144` as root.

> 
> Currently elasticsearch is being mapped to port 9200

Well, ES by default uses two ports, 9200 for the rest API and 9300 for internal stuff like this one.

File: docker-compose-build.yml (hunk 1 above)
Comment (sanacl) [reply docker-compose-build.yml]: Data can be lost in ES then. :thinking:

**Comment by sanacl** [comment]
@MarcioMateus I've been told I should not propose a PR against this branch. Sorry for this. I'll change the PR to be merged with the branch `bit`.

**Pull request closed without merging**